test/exchangeIssuance/exchangeIssuancePerp.spec.ts

```diff
@@ -0,0 +1,623 @@
+import "module-alias/register";
```

Comment:
Tests look very nice, since we are so close to 100% coverage would be great to fix those last missing branches / statements that are not covered yet see: https://coveralls.io/builds/51036727/source?filename=contracts%2FexchangeIssuance%2FExchangeIssuancePerp.sol

Also It'd be really great to have some integration tests that are running on a forked network against production deployments. 
See the integration tests of the leveraged ei module for reference:
https://github.com/IndexCoop/index-coop-smart-contracts/blob/master/test/integration/ethereum/exchangeIssuanceLeveragedIntegration.spec.ts

Ideally these tests would run on an optimism fork since this is where this will be used. You can compare the ethereum / polygon integration  tests to see what might be necessary to set up a new test pipeline for that .

Reply:
https://github.com/IndexCoop/index-coop-smart-contracts/pull/111/commits/93f5305b380ed7d3f1eb1b252c3d9d3161f2e541